src/mixins/input.js

```diff
@@ -79,6 +79,11 @@ export default {
     __onClick (e) {
       this.focus()
       this.$emit('click', e)
+    },
+    __onPaste (e) {
+      this.focus()
+      this.$emit('paste', e)
+      this.__emit()
```

Comment:
Hi, Is this.__emit() here necessary? It actually makes the `@input/@change` be emitted twice.